### PR TITLE
Parmedic Hardsuits: Now with 100% more hats 

### PR DESCRIPTION
**Pull request by Nickvr628**
Last one had a bug, thankfully it was an easy fix. Isn't version control great? This one has the hats back (although some are of dubious quality).

See #8693 and #8504 for discussion on this PR.

:cl:
fix: Paramedic Hardsuit PRs no longer cause all other hats to be gone
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
The same argument I had in your previous PR's before still stands just in-case you're curious.

TL;DR:

Make the hardsuit a spacesuit instead with no armor values and no thick armor.
would be nice but not as important as the first one:

make it have 2 pieces instead of 1 so it isn't more desirable vs a 2 piece suit otherwise this becomes a high-value item to steal for others vs EVA suits. (Also stealing from EVA is more dangerous than stealing from paramedics)

**Comment by Nickvr628** [comment]
While your ideas are appreciated, it does not align with my views for the desired end state of this PR. Furthermore as it was previously approved and merged as-is, I see any changes at this point detrimental to the mergability of this PR.

**Comment by maxion12345** [comment]
Justml merge it please I want my hats please

**Comment by Hopekz** [comment]
> Justml merge it please I want my hats please

This PR doesn't fix hats as it the changelog is false.

**Comment by Nickvr628** [comment]
Hopekz you absolute buffoon, go check how hardsuit code works.

**Comment by Partheo** [comment]
@Hopekz 
image

**Comment by Hopekz** [comment]
> @Hopekz
> image

I see; thanks!

**Comment by ThatLing** [comment]
I missed the original PR but why is this a hardsuit with such high resistances?

**Comment by SomeguyManperson** [comment]
you also apparently didn't add the no flash protect thing mentioned for it, or the lowered/no slowdown unless I'm missing something

**Comment by Hopekz** [comment]
> I see any changes at this point as detrimental to the mergability of this PR.

No PR bypasses the review process. It is up for review and changes just like any other PR.

**Comment by Hopekz** [comment]
Do paramedics wear bullet proof vest where you're from?
Why does a paramedic need thick armor and high armor stats?

**Comment by Nickvr628** [comment]
Yes paramedics do wear body armor where I live. This PR hardly gives them “high armor stats”. It is more so they can take a few more hits from things like space carp without dying.

**Comment by Nickvr628** [comment]
This was already approved and merged with both suits. Why are you making me change it now?

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -865,3 +865,25 @@
 	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 25)
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/powerarmor_advanced
 	item_color = "advancedpa1"
+
+//Paramedic Hardsuit for the yogstation specific paramedic role
+
+obj/item/clothing/head/helmet/space/hardsuit/paramedic
```
Comment (alexkar598) [inline code/modules/clothing/spacesuits/hardsuit.dm:871]: Missing a slash

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -865,3 +865,25 @@
...
+    desc = "A special helmet designed for work in a hazardous, low pressure environment. Built with lightweight materials for extra comfort, but does not protect the eyes from intense light."
+    alternate_worn_icon = 'icons/mob/head.dmi'
+    icon = 'icons/mob/head.dmi'
+    icon_state = "hardsuit0-para"
+    item_state = "para"
+    item_color = "para"
+    armor = list("melee" = 35, "bullet" = 10, "laser" = 15, "energy" = 5, "bomb" = 10, "bio" = 0, "rad" = 30, "fire" = 30, "acid" = 50)
+
+/obj/item/clothing/suit/space/hardsuit/paramedic
+    name = "Paramedic Hardsuit"
```
Comment (alexkar598) [inline code/modules/clothing/spacesuits/hardsuit.dm:882]: Should be all lowercase

**Comment by Nickvr628** [comment]
yeah I have lost most of my motivation to do this. Someone else can finish this if they want to. Every time we debate something and reach a conclusion it seems the (sometimes same) maintainer wants me to change or redo it the next day. 

Reverting the entire PR and making me redo all this instead of merging the perfectly acceptable bugfix was especially shitty.

**Pull request closed without merging**